guides/moving-to-v2.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Moving to Coder v2 Beta"
+description: What you need to know about Coder v2 
+---
+
+Coder v2 (also referred to as [Coder OSS](https://github.com/coder/coder))
+is Coder's open core remote development platform first launched in June 2022.
+This document shares best practices for moving your workflows from Coder v1 to
+Coder v2.
+
+> If you are current Coder v1 customer and to try Coder v2, [we'd like to hear from
+> you](https://calendly.com/bpmct/30min) to help inform our roadmap and
+> migration strategy.
+
+## Key concepts
+
+Coder v2 introduces a number of new paradigms. We recommend reading the comparison table before you proceed.
+
+| | Coder v1 | Coder v2 |
+| - | - | - |
+| **Workspace** | Each user creates and develops on remote workspaces | Same as Coder v1 |
+| **Supported IDEs** | Web IDEs (code-server, Jupyter) + SSH-powered desktop IDEs (e.g. VS Code, Jetbrains) | Same as Coder v1 |
+| **Provisioner** | Provisions workspaces on Kubernetes with hardcoded spec (pod + home volume) | Provisions workspaces via [Terraform](https://terraform.io). Supports any resource (e.g. Windows VM, Kubernetes pod) | 
+| **Template** | Optional YAML [configuration syntax](https://coder.com/docs/coder/latest/admin/templates) for workspaces. Managed by Coder admins or git/CI | [Terraform code](https://coder.com/docs/coder-oss/latest/templates) that defines workspace specs. Managed by Coder admins or git/CI | 
+| **Image** | Container image for workspace, contains dev tools and dependencies | Often included in [the template](https://coder.com/docs/coder-oss/latest/templates) |
```

Comment:
Container and VM images included in ....

Reply:
Can you clarify what you mean here?